examples/with-firebase-authentication/utils/auth/useUser.js

```diff
@@ -17,8 +22,6 @@ const useUser = () => {
       .signOut()
       .then(() => {
         // Sign-out successful.
-        cookies.remove('auth')
-        setUser()
```

Comment:
This happens automatically via the `onIdTokenChanged` below, so no need to duplicate it here.